sdk/python/kfp/client/client.py

```diff
@@ -176,7 +176,8 @@ def __init__(
             header_value=self._context_setting.get(
                 'client_authentication_header_value'))
         _add_generated_apis(self, kfp_server_api, api_client)
-        self._job_api = kfp_server_api.JobServiceApi(api_client)
+        self._recurring_run_api = kfp_server_api.RecurringRunServiceApi(
+            api_client)
```

Comment:
Per offline conversation, consider aliasing existing uses of "job" in the client's public API to use "recurring_run" instead if "recurring run" is the new canonical name for this object.